WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -84,6 +85,17 @@ private extension ProductFormViewController {
 //
 private extension ProductFormViewController {
     @objc func updateProduct() {
+        let title = NSLocalizedString("Publishing your product...", comment: "Title of the in-progress UI while updating the Product remotely")
+        let message = NSLocalizedString("Please wait while we publish this product to your store",
+                                        comment: "Message of the in-progress UI while updating the Product remotely")
+        let viewProperties = InProgressViewProperties(title: title, message: message)
+        let inProgressViewController = InProgressViewController(viewProperties: viewProperties)
+        if #available(iOS 13.0, *) {} else {
+            inProgressViewController.modalPresentationStyle = .overCurrentContext
```

Comment:
great catch! this `modalPresentationStyle` setting is for iOS 12, otherwise the view controller with a transparent background color will be presented and then the background turns dark.

for the safe area issue, it's from the blur view pinning to the safe area instead of the view's edges. I updated the xib in https://github.com/woocommerce/woocommerce-ios/pull/1679/commits/dc731356979cb77f87c137126f08c81234024aa0